src/uwtools/resources/jsonschema/upp.jsonschema

```diff
@@ -0,0 +1,188 @@
+{
+  "properties": {
+    "upp": {
+      "additionalProperties": false,
+      "properties": {
+        "execution": {
+          "$ref": "urn:uwtools:execution"
+        },
+        "files_to_copy": {
+          "$ref": "urn:uwtools:files-to-stage"
+        },
+        "files_to_link": {
+          "$ref": "urn:uwtools:files-to-stage"
+        },
+        "namelist": {
+          "additionalProperties": false,
+          "anyOf": [
+            {
+              "required": [
+                "base_file"
+              ]
+            },
+            {
+              "required": [
+                "update_values"
+              ]
+            }
+          ],
+          "properties": {
+            "base_file": {
+              "type": "string"
+            },
+            "update_values": {
+              "additionalProperties": false,
+              "properties": {
+                "model_inputs": {
```

Comment:
Are any of these properties `required` under `model_inputs`?

Reply:
Same answer as for `nampgb` :)